### PR TITLE
tell the dog to "sit" on drafts, etc.

### PR DESCRIPTION
**Pull request by ann-kilzer**
Signed-off-by: Ann Kilzer キルザー杏 <akilzer@gmail.com>

### CODE REVIEW

File: .github/workflows/reviewdog.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: reviewdog
 on: 
-  [pull_request]
+  pull_request:
+      types: [review_requested, ready_for_review]
```
Comment (ann-kilzer) [inline .github/workflows/reviewdog.yml:4]: Could also take review_requested off. What do you think @theyokohamalife

**Review by ermish — APPROVED** [review]
LGTM

**Comment by ermish** [comment]
also, that title XD

File: .github/workflows/reviewdog.yml (hunk 1 above)
Comment (ann-kilzer) [reply .github/workflows/reviewdog.yml:4]: If we add back code owners maybe it's good to take off `review_requested`

Comment (theyokohamalife) [reply .github/workflows/reviewdog.yml:4]: I think it would be a good idea to add code owners back now that the repo is public.

**Comment by theyokohamalife** [comment]
THANK YOU! The dog was going to drive me crazy 🤣

**Review by theyokohamalife — APPROVED** [review]

File: .github/workflows/reviewdog.yml (hunk 1 above)
Comment (ann-kilzer) [reply .github/workflows/reviewdog.yml]: It's so simple I am just making a PR and no issue: https://github.com/ourjapanlife/findadoc-server/pull/68

**Pull request merged**